.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
 language: ruby
 rvm:
   - 2.4.3
```

Comment:
Ruby 2.4.4 has also been released.
https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-4-4-released/

Is it intentional not to update this?

Reply:
No. Just overlooked. You can go ahead to open a pull request.

Reply:
I see. I will open a PRs for master and release52 branch later.